src/test/java/AuthenticationUtilityTest.java

```diff
@@ -0,0 +1,128 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import com.google.sps.utility.AuthenticationUtility;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+
+/** Test Authentication Utility functions TODO: Mock the Google verification service (Issue #5) */
+@RunWith(JUnit4.class)
+public final class AuthenticationUtilityTest {
+
+  // Will mock an HttpServletRequest to be passed to utilities
+  private final HttpServletRequest request = Mockito.mock(HttpServletRequest.class);
+
+  private final Cookie[] correctCookies =
+      new Cookie[] {
+        new Cookie("junk", "junk_Value"),
```

Comment:
nit: v in `junk_Value` here is capitalized, as well as in line 40.

Reply:
Thanks for finding this! Fixing